frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/DemoGraphTableFactory.java

```diff
@@ -0,0 +1,12 @@
+package com.virtuslab.gitmachete.frontend.ui.impl.table;
+
+import com.intellij.ui.table.JBTable;
+
+import com.virtuslab.gitmachete.frontend.ui.api.table.IDemoGraphTableFactory;
+
+public class DemoGraphTableFactory implements IDemoGraphTableFactory {
+  @Override
+  public JBTable create() {
```

Comment:
Since it's always returning the very same cached instance, maybe do rename `create` to `getInstance` and `(I)DemoGraphTableFactory` to `(I)DemoGraphTableProvider`